doc/user-guide/running.md

```diff
@@ -129,3 +139,34 @@ _ = execute_notebook(
     progress_bar=False,
 )
 ```
+
+## Removing cells
+
+```{versionadded} 0.0.21
+```
+
+```{admonition} Command-line equivalent
+:class: dropdown
+
+`ploomber-engine nb.ipynb output.ipynb --remove-tagged-cells remove`
+```
+
+If there are notebooks you want to remove before execution, tag them and use `remove_tagged_cells`. [This sample notebook](https://github.com/ploomber/ploomber-engine/blob/main/examples/remove.ipynb) contains one cell that will fail, if executed; however, the cell contains the tag `"remove"`, so let's remove it before execution:
```

Comment:
This should be `If there are cells you want to remove ` ?

Reply:
good catch! fixed it